frontend/micro-ui/web/micro-ui-internals/packages/modules/pgr/CHANGELOG.md

```
@@ -1,6 +1,64 @@
 # Changelog
 All notable changes to this module will be documented in this file.
 
+# Changelog
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)